doc/main.html

```diff
@@ -218,10 +277,11 @@ <h4>Scope</h4>
       </section>
 
       <section id="sec-conformance-section">
-        <h4>Conformance</h4>
+        <h4>Conformance section</h4>
 
-        <p>The <dfn>conformance section</dfn> contains conformance requirements and definitions, beyond the usual SMPTE conformance
-        statements.</p>
+        <p>The <dfn>Conformance</dfn> contains conformance requirements and definitions.</p>
```

Comment:
_The conformance_ does not read well.

Why did you change it from _the conformance section_?

Reply:
Oops. I removed section from the definitions (as shown in the table), didn't mean to remove from the descriptions. Updated.